packages/shared/src/components/post/freeform/write/WriteFreeformContent.tsx

```diff
@@ -16,21 +19,44 @@ export interface WriteFreeformContentProps {
   post?: Post;
 }
 
+interface WriteForm {
+  title: string;
+  content: string;
+  image: string;
+}
+
+export const generateWritePostKey = (reference = 'create'): string =>
+  `write:post:${reference}`;
+
 export function WriteFreeformContent({
   onSubmitForm,
   isPosting,
   squadId,
   post,
 }: WriteFreeformContentProps): ReactElement {
+  const formRef = useRef<HTMLFormElement>();
   const { shouldShowCta, isEnabled, onToggle, onSubmitted } =
     useNotificationToggle();
+  const key = generateWritePostKey(post?.id);
```

Comment:
There's a potential issue here that the storage will get flooded in case I edit many posts.
We have to remember to clean it if I publish the changes or discard them at some point